### PR TITLE
#178279113 load appropriate info for getAll referrals

### PR DESCRIPTION
**Pull request by tobieeniafe**
#### What does this PR do?
Load appropriate info for getAll referrals

#### Description of Task to be completed?
Load appropriate info for getAll referrals

#### How should this be manually tested?
`GET /api/v1/referral/all`

#### What are the relevant pivotal tracker stories?
#178279113

### CODE REVIEW

**Review by andela-hpopoola — CHANGES_REQUESTED** [review]
Thank you for working on this. This is getting into shape, we should be able to merge it after you add tests.

File: test/services/referral.service.test.js [hunk 1]
```diff
@@ -227,51 +225,4 @@ describe('Referral Service', () => {
       expect(ref._id).to.eql(referralId);
     });
   });
-
-  describe('#getUserByRefCode', () => {
```
Comment (andela-hpopoola) [inline test/services/referral.service.test.js:231]: Not sure why you removed this test. Can you add it back? We should focus on adding more tests not removing existing ones 🤷🏽‍♂️

**Review by andela-hpopoola — APPROVED** [review]
Thank you for working on this. Looks good to me. Can you add the removed service test before you merge 🙏🏽 ? Thank you and Nice work.

File: test/services/referral.service.test.js (hunk 1 above)
Comment (tobieeniafe) [reply test/services/referral.service.test.js:231]: It was removed in error. My bad 🤦🏾‍♂️

**Pull request merged**